src/utils/localStorage.js

```diff
@@ -0,0 +1,16 @@
+export function getLocalStorageItem(key) {
+    try {
+      return JSON.parse(localStorage.getItem(key));
+    } catch (error) {
+      console.log(error);
+      return null;
+    }
+  }
+  
+  export function setLocalStorageItem(key, value) {
+    localStorage.setItem(key, JSON.stringify(value));
```

Comment:
You can also catch the exception here - in some cases, you can get this: `Setting the value of '<key>' exceeded the quota` which basically informs you that there is no space left in local storage (or browser memory allocated for it) - it's just one example. That's not in the scope of this particular project, because we probably won't need to store items that large, but it's good to know.